src/Commands/pm/SecurityUpdateCommands.php

```diff
@@ -7,6 +7,7 @@
 use Drush\Commands\DrushCommands;
 use Drush\Drush;
 use Exception;
+use function str_replace;
```

Comment:
er, i have not seen a regular function in a use statement. i have seen functions prefixed with a slash to indicate that it is a global function. lets do that if needed.